.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules
 dist
 .DS_Store
-coverage
\ No newline at end of file
+coverage
+package-lock.json
```

Comment:
Any specific reason to include `package-lock` in ignore list? One of CI job has been failing after this.
![image](https://user-images.githubusercontent.com/883233/95023684-3f4c5d00-0633-11eb-84fd-df78e62ca4b2.png)

https://stackoverflow.com/a/44210813
https://stackoverflow.com/a/48524475